IABot/checkIfDead.php

```diff
@@ -58,23 +58,22 @@ public function checkDeadlinks( $urls ) {
 			curl_setopt( $curl_instances[$id], CURLOPT_NOBODY, 1 );
 			curl_setopt( $curl_instances[$id], CURLOPT_RETURNTRANSFER, true );
 			curl_setopt( $curl_instances[$id], CURLOPT_FOLLOWLOCATION, true );
-			curl_setopt( $curl_instances[$id], CURLOPT_TIMEOUT, 30 ); // Set 30 seconds timeout for the entire curl operation to take place
+			curl_setopt( $curl_instances[$id], CURLOPT_TIMEOUT, 60 ); // Set 60 second timeout for the entire curl operation to take place
```

Comment:
I thought about that but it seemed way too complicated without any obvious benefits. If you want, feel free to refactor the code to reuse the same function. Don't forget to add a base case for the recursion.